_episodes/11-software-project.md

```diff
@@ -55,67 +55,59 @@ to fix and build on top of the existing code during the course.
 ## Downloading Our Software Project
 
 To start working on the project, you will first create a fork of the software project repository
-from GitHub within your own GitHub account
-and then obtain a local copy of that project (from your GitHub) on your machine.
+from GitLab within the workshop's GitLab group
+and then obtain a local copy of that project (from your GitLab) on your machine.
 
-1. Make sure you have a GitHub account
-   and that you have [set up SSH key pair for authentication with GitHub](../setup.html#secure-access-to-github-using-git-from-command-line).
+1. Make sure you have a GitLab account
+   and that you have [set up SSH key pair for authentication with GitLab](../setup.html#secure-access-to-gitlab-using-git-from-command-line).
    
    ***Note:** while it is possible to use HTTPS with a personal access token for authentication
-   with GitHub, the recommended and supported authentication method to use for this course 
+   with GitLab, the recommended and supported authentication method to use for this course 
    is SSH with key pairs.*
-2. Log into your GitHub account.
-3. Go to the [software project repository](https://github.com/carpentries-incubator/python-intermediate-inflammation)
-   in GitHub.
-
-   ![Software project fork repository in GitHub](../fig/github-fork-repository.png){: .image-with-shadow width="900px" }
-
+2. Log into your GitLab account.
+3. Go to the [software project repository](https://gitlab.com/svenvanderburg/python-intermediate-inflammation)
+   in GitLab.
 4. Click the `Fork` button
-   towards the top right of the repository's GitHub page to **create a fork** of the repository 
-   under your GitHub account.
-   Remember, you will need to be signed into GitHub for the `Fork` button to work.
+   towards the top right of the repository's GitLab page to **create your own fork** of the repository 
+   under the workshop's GitLab group.
+   Remember, you will need to be signed into GitLab for the `Fork` button to work.
 
    ***Note:** each participant is creating their own fork of the project to work on.*
 
    ***Note 2:** we are creating a fork of the software project repository (instead of copying it 
    from its template) because we want to preserve the history of all commits (with template copying 
    you only get a snapshot of a repository at a given point in time).*
-5. Make sure to select your personal account
-   and set the name of the project to `python-intermediate-inflammation`
+5. Make sure to select the workshop GitLab group as the **namespace**
```

Comment:
Is this going to work when multiple participants fork the repo into the same namespace?

Reply:
Yes if they add their name to the end!